### ISSUE TITLE
TypeError: path must be a string

### ISSUE THREAD

**Issue by vovcat** [harthur/replace#26]
When file name (in this example - "1") consists of decimal digits, replace crashes:

```
(shell) $ replace.js a b 1 -p

fs.js:679
  return binding.lstat(pathModule._makeLong(path));
                 ^
TypeError: path must be a string
    at Object.fs.lstatSync (fs.js:679:18)
    at replacizeFileSync (replace/replace.js:110:22)
    at module.exports (replace/replace.js:53:13)
    at Object.<anonymous> (replace/bin/replace.js:55:1)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)
```

**Comment by vovcat** [comment]
That can be cured by:

```
diff --git bin/replace.js bin/replace.js
index ccd7e0d..25e251a 100755
--- bin/replace.js
+++ bin/replace.js
@@ -15,6 +15,7 @@ var addlOptions = {
         position: 2,
         help: "File or directory to search (default is '*')",
         list: true,
+        type: "string",
         default: ["*"]
     },
     funcFile: {
```

**Comment by harthur** [comment]
Thanks for filing.

**Comment by harthur** [comment]
Should be fixed with 89b3754
